src/pem/_core.py

```diff
@@ -40,6 +40,16 @@ def as_bytes(self):
         """
         return self._pem_bytes
 
+    def __eq__(self, other):
+        return (type(self) == type(other) and
```

Comment:
Yeah I guess it’s convenient to be able to compare all our types with each other…